<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertFinishedDialog</class>
 <widget class="QDialog" name="InsertFinishedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Finished</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>The insert has finished. Copy the request key from below:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InsertFinishedDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
